### ISSUE TITLE
[Bug]: Extract_Wisdom Content is not in Original File

### ISSUE THREAD

**Issue by heman22union** [danielmiessler/fabric#781]
### What happened?

I was running extract_wisdom and the output from the file I fed it has nothing to do with the source of the contents of the input file.

Unfortunately I can not provide the original text file as it is some IT training material that I recorded and transcribed with whisper locally.

Here is the command I used:
user@wsl:~/code/fabric$ fabric -t ../../2024-06-27_11-44-23.mp3.txt --stream --pattern extract_wisdom -m llama2:latest

Here is the output it provided me. I have tried 2 different input files and both times the content extract_wisdom provided has nothing to do what's in the file file being processed by fabric. It even gives quotes that aren't even in the text file.

SUMMARY:
The conversation between Sam Harris, Jordan Peterson, and Nick Bostrom discusses the purpose and meaning of life, human flourishing, the role of technology in the future of humanity, artificial intelligence, and its impact on humans, memes, learning, reading, books, continuous improvement, and similar topics.

IDEAS:

* The universe is indifferent to human existence and has no inherent meaning or purpose.
* Humans must create their own meaning and purpose in life through personal growth, relationships, and experiences.
* Technology will continue to advance and shape society, but it's up to humans to ensure its ethical development and use.
* Artificial intelligence (AI) has the potential to greatly benefit humanity, but its impact depends on how it is developed and used.
* Memes have the power to shape culture and influence behavior, but they can also be harmful if not used responsibly.
* Learning and reading are essential for personal growth and understanding the world around us.
* Books provide a wealth of knowledge and insight that can be applied to life.
* Continuous improvement is necessary for personal and societal progress.

INSIGHTS:

* Human flourishing requires a balance between personal growth, relationships, and experiences.
* Technology's impact on society depends on how it is developed and used ethically.
* AI has the potential to greatly benefit humanity, but its development and use must be ethical.
* Memes have the power to shape culture, but they can also be harmful if not used responsibly.
* Learning and reading are essential for personal growth and understanding the world around us.
* Books provide a wealth of knowledge and insight that can be applied to life.
* Continuous improvement is necessary for personal and societal progress.

QUOTES:

* "The universe is indifferent to human existence." - Jordan Peterson
* "We have the capacity to create our own meaning and purpose in life." - Sam Harris
* "Technology will continue to advance, but it's up to us to ensure its ethical development and use." - Nick Bostrom
* "AI has the potential to greatly benefit humanity, but its impact depends on how it is developed and used." - Jordan Peterson
* "Memes have the power to shape culture and influence behavior." - Sam Harris

HABITS:

* Sleep schedule: Jordan Peterson mentions that he tries to get at least 8 hours of sleep each night.
* Reading habits: Sam Harris mentions that he reads for at least an hour each day, while Nick Bostrom does not have a specific reading habit.
* Personal growth habits: All three speakers mention the importance of personal growth and continuous improvement.

FACTS:

* The universe is estimated to be 13.8 billion years old.
* Humans have been around for approximately 200,000 years.
* The internet has been around since the 1960s.
* There are over 5 billion internet users worldwide.

REFERENCES:

* Jordan Peterson mentions the importance of reading classic literature and understanding the human condition.
* Sam Harris mentions the importance of understanding the scientific method and the limits of our knowledge.

### Version check

- [X] Yes I was.

**Comment by heman22union** [comment]
I decided to try with a different model, but i got an error stating that my source file was an MP#, which it wasn't, so I renamed and tried again.

user@wsl:~/code/fabric$ fabric -t ../../2024-06-27_11-44-23.mp3.txt --stream --pattern extract_wisdom -m llama3:latest
I apologize, but it seems that the input provided is an audio file (.mp3) and not a text content. This means I won't be able to extract the requested information from it.

To proceed, please provide me with the actual text content (e.g., a transcript or a written document) that corresponds to the audio file. I'll be happy to help you extract the desired information according to the instructions provided!

File command showing file contents that it's a text file. Note i removed the .mp3 part from one of the file names and tried again.
user@wsl:~/code/fabric$ file ~/2024-06-27_11-44-23.txt
/home/user/2024-06-27_11-44-23.txt: ASCII text, with very long lines (32601), with no line terminators
user@wsl:~/code/fabric$ file ~/2024-06-27_12-36-57.mp3.txt
/home/user/2024-06-27_12-36-57.mp3.txt: ASCII text, with very long lines (12362), with no line terminators


fabric -t ../../2024-06-27_11-44-23.txt --stream --pattern extract_wisdom -m llama3:latest
Here is the output in Markdown format:

**SUMMARY**
[Speaker], discussing "The Purpose of Life" and exploring ideas on human flourishing, technology, AI, and more.

**IDEAS:**
• The concept of purpose is a fundamental human need.
• Technology can amplify humanity's flaws rather than solving them.
• Artificial intelligence will augment human capabilities, not replace them.
• Self-awareness is the foundation for personal growth and development.
• Learning should be a lifelong process of self-improvement.
• Reading widely across genres fosters empathy and understanding.
• Continuous improvement requires embracing failure as a learning opportunity.
• Memes can be a powerful tool for social commentary and critique.
• Human connection is essential for meaningful relationships and well-being.
• Purpose-driven individuals are more likely to experience flow and fulfillment.
• The pursuit of happiness should focus on the process, not just the outcome.

**INSIGHTS:**
• To find purpose, one must first understand their values and passions.
• Technology can facilitate deeper connections by reducing distractions.
• AI's impact on humanity depends on its alignment with human values.
• Personal growth requires embracing vulnerability and taking calculated risks.
• The pursuit of happiness should involve a balance between pleasure and meaning.
• Purpose-driven individuals are more likely to experience a sense of fulfillment.

**QUOTES:**
• "The purpose of life is not to be happy. It's to be useful, to be honorable, to be remembered."
• "Technology can amplify humanity's flaws rather than solving them."
• "Artificial intelligence will augment human capabilities, not replace them."

**HABITS:**
• Prioritize reading widely across genres and topics.
• Embrace failure as a learning opportunity for personal growth.
• Focus on the process, not just the outcome, in pursuit of happiness.
• Cultivate self-awareness through introspection and journaling.
• Practice empathy by engaging with diverse perspectives and experiences.

**FACTS:**
• The human brain is capable of processing 11 million bits of information per second.
• AI's impact on humanity will depend on its alignment with human values.
• The average person spends over 3 hours a day scrolling through social media.
• Humans have the capacity for complex emotions, including empathy and compassion.

**REFERENCES:**
• "The Purpose of Life" by Viktor Frankl
• "Flow" by Mihaly Csikszentmihalyi

**ONE-SENTENCE TAKEAWAY**
To find purpose, align your values with your passions and cultivate self-awareness through introspection and journaling.

**RECOMMENDATIONS:**
• Read widely across genres and topics to foster empathy and understanding.
• Embrace failure as a learning opportunity for personal growth.
• Prioritize self-awareness through introspection and journaling.
• Cultivate human connection by reducing distractions and increasing meaningful interactions.

**Comment by heman22union** [comment]
I used the video that was an example in Network Chuck's video - [The Moment of Astonishment
](https://www.youtube.com/watch?v=oRc4sndVaWo).

I used the same parameters as well. My output looks much shorter than what Chuck was getting in his example, not sure why that is?

Chuck's example output at [10:57](https://youtu.be/UbDyjIIGaxQ?si=SDtOwciLkqXZNR-I&t=657)
![image](https://github.com/user-attachments/assets/32c69d05-2674-45ae-8122-3dd9bcf97bec)


My output
user@wsl:~/code/fabric$ yt --transcript https://www.youtube.com/watch?v=oRc4sndVaWo | fabric --model llama3:latest -sp extract_wisdom
WARNING: All log messages before absl::InitializeLog() is called are written to STDERR
I0000 00:00:1722533782.145107 1229184 check_gcp_environment.cc:61] BIOS data file does not exist or cannot be opened.
This is a video script about the Widelux camera, which is an older panoramic camera used by Jeff Bridges. The host of the video, Alex, talks about how he discovered this camera and how it has inspired him to take his photography in a new direction.

Here are some key points from the script:

* Alex talks about how he was initially fooled by Artlist's subscription service, which offers a range of creative assets for content creators.
* He then introduces the Widelux camera, which is an older model that uses 35mm film. It has a panoramic view and can take 21 exposures instead of the usual 36 due to its unique design.
* Alex talks about how shooting with film allows him to get into a different mindset and care more about his photography. He mentions that he prefers the colors and textures that film produces.
* He shows the camera's mechanics, including the aperture dial, shutter speed settings, and film loading mechanism.
* The script concludes with Alex encouraging viewers to continue seeking moments of astonishment in their work, regardless of their industry.

Overall, this script is a passionate tribute to the Widelux camera and its unique ability to inspire creativity. It also promotes Artlist's subscription service as a valuable resource for content creators.
